### ISSUE TITLE
Logging

### ISSUE THREAD

**Issue by martinezmatias** [SpoonLabs/flacoco#105]
When I change the log level in Astor to Debug, Flacoco (or TestRunner) prints the coverage results.
The large amount of info printed produces that I cannot continue with the debugging.

The problem is in: 
```DEBUG CoverageRunner OnlineCoveredTestResultPerTestMethodImpl```

**Comment by andre15silva** [comment]
This is true for large projects, since we output the entire coverage result per test at the debug level.

We should, instead, output it to a file.
